GOVERNANCE.md

```diff
@@ -244,7 +277,153 @@ The current NumFOCUS Subcommittee consists of:
 - Jon Sedar
 - Thomas Wiecki
 
-### Institutional Partners and Funding
+## Core Contributors
+Core Contributors are those individuals with commit or managing rights to any of
+the repositories or services of the PyMC Project. However, the exact permissions
+of all Core Contributors may not be the same and depend on their team memberships.
+Therefore, Core Contributors may act as Recurrent Contributors depending
+on the task at hand.
+We believe this to be necessary given the current size and scope of
+the PyMC project. Contributors may be proficient in some areas of the
+project while being inexperienced in some others.
+
+### Core Contributor membership
+To become a Core Contributor, one must already be a Recurring Contributor.
+Core Contributors can nominate any Recurring Contributor to become a
+Core Contributor. For the nomination to go forward, it has to be
+ratified by the Steering Council and the BDFL.
+For a nomination to be rejected, clear reasoning behind the decision must be
+shared with the rest of the team. People whose nomination has been rejected can
+be nominated at any time again in the future, three months after the previous
+nomination at the earliest.
+
+### Current Core Contributors
+
+<!-- we could list them here or link to the pymc-devs organization, however,
+linking to the pymc-devs org would require the core team not being secret.
+I actually don't know if this is already the case, if it's possible even nor
+obviously how it would be done. I do believe it's important to be clear and
+explicit about who are core contributors, especially for people interested
+in joining the project to reach out, but also for organizations and meet-ups
+to ask for speakers.
+
+Regarding the reaching out, we do have the project mail and some people
+may not desire being contacted privately. I also think we should have some
+rules or guidance on this here and/or in the community/contributing pages.-->
+
+## Recurring Contributors
+Recurring Contributors are those individuals who can participate from
+The Project private communication channels.
+
+### Recurring Contributor membership
+Recurring Contributors can nominate any Contributor to participate in the
+Project private communication channels and become a Recurring Contributor.
+For the nomination to go forward, it has to be ratified by
+the Steering Council and the BDFL.
+For a nomination to be rejected, clear reasoning behind the decision must be
+shared with the rest of the team. People whose nomination has been rejected can
+be nominated at any time again in the future, three months after the previous
+nomination at the earliest.
+
+#### Current Recurring Contributors
+
+- everyone on slack! This may be too long so it could be a good idea to
+format as a table instead or to put within a `<details>` tag so
+it's hidden by default while still available publicly
```

Comment:
Three levels of membership (council/core/recurring) are a lot to manage already. I don't think we need to keep a list of recurring contributors, since there are no formal rights/obligations with that status. (Other than being on the Slack channel?)

Reply:
While recurrent contributors don't have many rights nor obligations by design<sup>1</sup> one of the goals in "creating" the position is making new contributors feel and be part of the team earlier on with the hopes of attracting and retaining more people, and one general goal of the governance update was to be more clear and transparent.

I'd suggest we try having this here and if at some point it gets outdated or is too much work for us.

1: they only participate from private channels, slack and lab meetings now, and can nominate people for recurrent contributors, but they could also vote steering council members even if they were not eligible :thinking:

Reply:
I agree that tracking recurring contributors is not something we should invest much (any) time in.

Reply:
I think especially for recurrent contributors it will be important to a lot of people and will help new members feel part of the team more easily. I suggest we try it (we can make new members add themselves here so we only have to merge a PR) and see if it becomes too much work.

Reply:
i’d say it’s worth trying because i don’t know with all the new people joining slack in docs team etc, we can make them feel like they’re a real part of the team and not guests this way. i think we can find a middle ground. the goal remains: make it easier for people to contribute + accessible/welcoming to join the community. so i would say we do the safe and easier thing of: once people contribute and we think they'll be helpful in discussions we invite some of them to slack -> if we see that they contribute for n number of weeks more (this can be informal, say 3-4 weeks or 2-3 important PRs) then we give them triage permissions. so they're a part of the organization, but because they're new people they don't get write permissions. i also agree with oriol's idea "I suggest we try it (we can make new members add themselves here so we only have to merge a PR) and see if it becomes too much work." this sounds good as well. either way, the exact method doesnt matter, we dont need to formally track them, but just things we can do to make people feel like an active part of the community. like it was after chris invited me to slack that i got invested in discussions and meetings and i think we would want that, people who are genuinely invested in pymc